schism/compiler.ss

```diff
@@ -922,7 +934,7 @@
         `(,tag ,(cadr expr) . ,(annotate-free-vars-expr* (cddr expr) bodies)))
        ((eq? tag 'apply-procedure)
         `(apply-procedure . ,(annotate-free-vars-expr* (cdr expr) bodies)))
-       ((or (eq? tag 'var) (eq? tag 'nop) (eq? tag 'i32) (eq? tag 'const))
+       ((memq tag '(top var nop i32 const))
```

Comment:
Nice cleanup!

Reply:
Seems like a `case` form might be nice with all the tag inspection!